tests/integration/links_test.go

```diff
@@ -36,6 +36,7 @@ func TestLinksNoLogin(t *testing.T) {
 		"/user2/repo1/",
 		"/user2/repo1/projects",
 		"/user2/repo1/projects/1",
+		"/user2/repo1/releases/tag/delete-tag",
```

Comment:
Why use "delete-tag" to test? A "deleted tag" should not exist literally?

Reply:
It's the only one existing record on release.yml which has `is_tag: true`.

Reply:
Then I think it needs a comment, to say that is really exists

Reply:
03a3e9881c